modules/drivers/mongo/src/metabase/driver/mongo/query_processor.clj

```diff
@@ -279,19 +293,16 @@
           ;; stringify it as yyyy-MM Subtracting (($dayOfYear(column) % 91) - 3) days will put you in correct month.
           ;; Trust me.
           :quarter
-          (mongo-let [#_:clj-kondo/ignore parts {:$dateToParts {:date column}}]
-            {:$dateFromParts {:year  :$$parts.year
-                              :month {$subtract [:$$parts.month
-                                                 {$mod [{$add [:$$parts.month 2]}
-                                                        3]}]}}})
+          (if date-trunc?
+            (truncate :quarter)
+            (mongo-let [#_:clj-kondo/ignore parts {:$dateToParts {:date column}}]
```

Comment:
You can do `#_{:clj-kondo/ignore [:unresolved-var]}` or whatever to ignore just a specific linter rather than everything